packages/relay/src/lib/eth.ts

```diff
@@ -1269,8 +1275,8 @@ export class EthImpl implements Eth {
         }
 
         if (e.isContractReverted()) {
-          this.logger.trace(`${requestIdPrefix} mirror node eth_call request encoutered contract revert. message: ${e.message}, details: ${e.detail}, data: ${e.data}`);
-          return predefined.CONTRACT_REVERT(e.detail || e.message, e.data);
+          this.logger.trace(`${requestIdPrefix} mirror node eth_call request encountered contract revert. message: ${e.message}, details: ${e.detail}, data: ${e.data}`);
+          return predefined.CONTRACT_REVERT(e.message + e.detail, e.data);
```

Comment:
Should not be something like this preferable:
```suggestion
          return predefined.CONTRACT_REVERT(`message: ${e.message}, detail: ${e.detail}`, e.data);
```
or
```suggestion
          return predefined.CONTRACT_REVERT(`${e.message} : ${e.detail}`, e.data);
```

Reply:
Updated.

Reply:
Actually the test 'SDK returns a precheck error' expects only the 'detail' in the error.  Updated to conform to the test.